### PR TITLE
Minor post-transfer changes

### PR DESCRIPTION
**Pull request by newpavlov**
Not sure about the minimum supported Rust version for CI, for now it's 1.27.

### CODE REVIEW

**Comment by newpavlov** [comment]
`key::tests::key_generation_128` has failed with "unwrap() on None". I've restarted the test and it got "fixed". IIUC you have non-determinism in your tests and implementation has a bug.

Ideally we should be able to use deterministic RNG source for tests and OS RNG for everything else, but unfortunately it's not that [easy](https://github.com/rust-random/rand/issues/648) right now.

**Comment by newpavlov** [comment]
BTW it would be nice to also add fuzzing testing, see [`cargo-fuzz`](https://github.com/rust-fuzz/cargo-fuzz).

**Comment by dignifiedquire** [comment]
> key::tests::key_generation_128 has failed with "unwrap() on None". I've restarted the test and it got "fixed". IIUC you have non-determinism in your tests and implementation has a bug.

Argh :/ I saw this once before in a project that uses this, but didn't realize it comes from here. 

Created an issue #2 


> BTW it would be nice to also add fuzzing testing, see [`cargo-fuzz`](https://github.com/rust-fuzz/cargo-fuzz).

Yes, definitely need this #3

File: Cargo.toml [hunk 1]
```diff
@@ -1,13 +1,13 @@
 [package]
 name = "rsa"
 version = "0.1.0"
-authors = ["dignifiedquire <dignifiedquire@gmail.com>"]
-description = "RSA implementation in Rust"
```
Comment (jarlungoodoo73) [inline Cargo.toml]: Delete

File: Cargo.toml [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+authors = ["RustCrypto Developers", "dignifiedquire <dignifiedquire@gmail.com>"]
+description = "Pure Rust RSA implementation"
 license = "MIT OR Apache-2.0"
-homepage = "https://github.com/dignifiedquire/rust-rsa"
```
Comment (jarlungoodoo73) [inline Cargo.toml]: Delete

File: Cargo.toml [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
-repository = "https://github.com/dignifiedquire/rust-rsa"
```
Comment (jarlungoodoo73) [inline Cargo.toml]: Delete

File: Cargo.toml [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
-
```
Comment (jarlungoodoo73) [inline Cargo.toml]: Delete

File: README.md [hunk 5]
```diff
@@ -1,6 +1,7 @@
 # RSA
+[![crates.io](https://img.shields.io/crates/v/rsa.svg)](https://crates.io/crates/rsa) [![Documentation](https://docs.rs/rsa/badge.svg)](https://docs.rs/rsa) [![Build Status](https://travis-ci.org/RustCrypto/RSA.svg?branch=master)](https://travis-ci.org/RustCrypto/RSA) [![dependency status](https://deps.rs/repo/github/RustCrypto/RSA/status.svg)](https://deps.rs/repo/github/RustCrypto/RSA)
 
-> A portable RSA implementation in Rust.
```
Comment (jarlungoodoo73) [inline README.md]: Delete

**Review by jarlungoodoo73 — APPROVED** [review]
Delete all red lines, green lines are all go, there the fix. Review, fix, finish

**Pull request merged**